packages/invoices/src/utils.ts

```diff
@@ -24,9 +24,9 @@ export const placeholderInbound: InboundShipment = {
   otherParty: undefined,
   otherPartyId: '',
   items: [],
-  status: InvoiceNodeStatus.Draft,
+  status: InvoiceNodeStatus.New,
   type: InvoiceNodeType.InboundShipment,
-  entryDatetime: '',
+  createdDatetime: '',
```

Comment:
these should be getting cast to a Date object, but that was going to create a bunch more changes..

Reply:
ah, ok. dates again huh?